ext/com_dotnet/com_wrapper.c

```diff
@@ -424,7 +424,7 @@ static void generate_dispids(php_dispatchex *disp)
 	zend_string *name = NULL;
 	zval *tmp, tmp2;
 	int keytype;
-	zend_long pid;
+	zend_ulong pid;
```

Comment:
Don't you have warning with unsigned/signed usage (e.g. line 461) ?

Reply:
No, but I might miss it (building with several warnings disabled, because there are so many). Let's leave this as is for now.